src/chain_spec.rs

```diff
@@ -744,25 +623,18 @@ pub mod alcyone_testnet {
         genesis(
             vec![get_authority_keys_from_seed("Alice", false)],
             seeded_acc_id("Alice"),
-            vec![
-                seeded_acc_id("Bob"),
-                seeded_acc_id("Bob//stash"),
-                seeded_acc_id("relay_1"),
-                seeded_acc_id("relay_2"),
-                seeded_acc_id("relay_3"),
-                seeded_acc_id("relay_4"),
-                seeded_acc_id("relay_5"),
-            ],
             true,
+            BridgeLockId::new(1, BRIDGE_LOCK_HASH),
+            BridgeLockId::generate_bridge_locks(20),
         )
     }
 
     pub fn develop_config() -> ChainSpec {
         // provide boot nodes
         let boot_nodes = vec![];
         ChainSpec::from_genesis(
-            "Polymesh Alcyone Develop",
-            "dev_alcyone",
+            "Polymesh Testnet Develop",
```

Comment:
Also modify https://github.com/PolymathNetwork/Polymesh/blob/50dfe42e9350c0334c5c813e0681112f1a5d5b3b/src/service.rs#L55 to accept "Polymesh Testnet".

Reply:
It already has "Polymesh Testnet" instead of "Polymesh Alcyone"
https://github.com/PolymathNetwork/Polymesh/pull/966/files#diff-5c6f87176b7f64fbbeabbc31f5ad377edc268ebeb231eb55ee31c14ec3834f23R55

Reply:
We need both `"Polymesh Alcyone"` and `"Polymesh Testnet"` because Alcyone is still called Alcyone in its config (hardcoded json).

Reply:
Alcyone config is stored in a JSON file and not as a module in `chain_spec.rs`. Meanwhile `testnet` config is only stored as a Rust module. Both cannot resolve to `Network::Testnet` at the same time because `Network::Testnet` correspond to the Rust module and therefore to `testnet`.